pandas/core/generic.py

```diff
@@ -4581,43 +4580,32 @@ def pipe(self, func, *args, **kwargs):
 
     Parameters
     ----------
-    func : callable, string, dictionary, or list of string/callables
-        To apply to column
+    func : function, string, list of string/functions or dictionary
+        Function to use for transforming the data. If a function, must either
+        work when passed a %(klass)s or when passed to %(klass)s.apply.
+        The function (or each function in a list/dict) must return an
```

Comment:
IMO would be better served in a Notes section along with description of what happens if something is returned that doesn't meet this requirement

Reply:
Hmm, it is already described in the Raises section, so I could just drop this part here...